### ISSUE TITLE
Spatial Audio futher support

### ISSUE THREAD

**Issue by Riccardo-Axed** [RenderHeads/UnityPlugin-AVProVideo#789]
Hi RenderHeads,

I've purchased the AVPro Ultra Edition to have support for the spatial audio, and it works as long as I strictly use Facebook Audio 360 with a mkv container.

However, it would be nice to have a fuller support for this feature - that is, mp4 files with Ambisonic audio (1st and 2nd order) - mainly because of the better support and standardization for the mp4 container in the multimedia industry.

Another useful feature would be to have AVPro automatically detect the audio 360 channel format (right now, I've to set it manually).

To summarize, I'd like AVPro could manage spatial audio in straight and transparent way, with a wider formats support and being able to automatically switch between ordinary audio and spatial audio, detecting the many channel formats by itself.

Thank you,

Riccardo

**Comment by Rod-Elevative** [comment]
Hello!

I would like to 1+ with the suggestion above to implement industry standard 1st and 2end order ambisonic audio formats. It would allow numerous other workflows to be used with other plugin suites when generating audio (DearVR, IEM, ect.).

The mkv container that FB360 used is actually a depreciated encoding feature and they recommend using ambisonics instead (its in their plugin package user manual on page 32). For new downloaders of the plugins its also currently not possible to get the 8ch decode format working on windows due to their software not being able to download the correct git of ffmpeg because the host server went down in Oct and hasn't been fixed (there's a bunch of threads out there about this).

If AmbiX can get implemented I would gladly throw my monies at AVPro Ultra xP

Best Regards,

- Jacob

**Comment by visjack** [comment]
I'd like to bump this too. Are there any plans for this?

**Comment by Voynaa** [comment]
> Hello!
> 
> I would like to 1+ with the suggestion above to implement industry standard 1st and 2end order ambisonic audio formats. It would allow numerous other workflows to be used with other plugin suites when generating audio (DearVR, IEM, ect.).
> 
> The mkv container that FB360 used is actually a depreciated encoding feature and they recommend using ambisonics instead (its in their plugin package user manual on page 32). For new downloaders of the plugins its also currently not possible to get the 8ch decode format working on windows due to their software not being able to download the correct git of ffmpeg because the host server went down in Oct and hasn't been fixed (there's a bunch of threads out there about this).
> 
> If AmbiX can get implemented I would gladly throw my monies at AVPro Ultra xP
> 
> Best Regards,
> 
> * Jacob

+1
